tools/babel/scripts/folder.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env sh
+
+# Copyright JS Foundation and other contributors, http://js.foundation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+echo Converting, "$1 to $2"!
+./node_modules/.bin/babel $1 --out-dir $2 --verbose
```

Comment:
I think we can combine these two script by checking the type of the argument. Also you should check the number of arguments before accessing $1 and $2.